framework/src/main/java/com/github/dbmdz/flusswerk/framework/reporting/ProcessReport.java

```diff
@@ -35,4 +35,9 @@
    * @param e The exception, why the message failed
    */
   void reportReject(Message message, Exception e);
+
+  default void reportSkip(Message message, Exception skip) {
+    reportSuccess(message);
```

Comment:
Maybe add a prefix to the message in the default implementation that this was not a 'proper' successful run?  Might be confusing otherwise!

Reply:
message is of type Message here, not String. The implementation of reportSucess is responsible for rendering a Message. I will add a Default implementation in DefaultProcessReport, though.